web_viewer/index.html

```diff
@@ -215,6 +263,8 @@
 
         function on_wasm_error(error) {
             console.error("Failed to start: " + error);
+
+            let render_backend_name = wasm_bindgen.is_webgpu_build() ? "WebGPU" : "WebGL";
```

Comment:
`on_wasm_error` is also called if the wasm fails to load, which would lead `wasm_bindgen.is_webgpu_build` to fail

Reply:
good catch. solved it with a try catch now